src/sentry/tasks/post_process.py

```diff
@@ -613,16 +613,25 @@ def process_code_mappings(job: PostProcessJob) -> None:
         # This will only process code mappings one project per org per hour but we can do better.
         if organization_queued is None:
             cache.set(cache_key, True, 3600)
-            logger.info(
```

Comment:
This line logs a line for every organization, regardless of whether the flag is enabled, which makes it difficult to find the important logs.